### DISCUSSION TITLE
How to go from https://modelviewer.dev/editor/ to HTML code to add to website

### DISCUSSION THREAD

**Discussion by JoelTre** [google/model-viewer#4117] [Q&A]
Hi there

I wish to get HTML code to allow people to view my model on my website using model-viewer. However, I have not been able to figure out how to do this after many hours and was going to give up (after trying to find previous answers on here) but thought I'd ask here for help. I was able to open my model in https://modelviewer.dev/editor/ and edit the materials as needed. However, I could not figure out how to go from this point to having HTML code that I can copy and paste into my website. I looked through a lot of HTML code on websites that use the viewer to attempt to figure out how they did it, but could not. 

Any help would be greatly appreciated. I think it would be great to have a step-by-step list of steps to take on the home website (https://modelviewer.dev/) so that someone like me can know how to use model-viewer. Even ChatGPT could not get the code working. I also deployed it to two different mobiles but that did not work. 

Cheers

Joel

**Comment by elalish** [comment]
Did you try the "Download Scene" button? It gives you a whole starter website. Or even more simply, did you try the two lines from https://modelviewer.dev/? Always check your JS console for errors, and you may find useful info in our [FAQ](https://modelviewer.dev/docs/faq.html).

**Reply by JoelTre** [reply]
Yes, I used the button and it gave me a zip folder with a lot of little bits of code but I could not figure out how to use each bit of code and where to paste them into the HTML code of a website. The different places I tried to paste the different bits but they never seemed to work. Where am I meant to paste each bit of code? 

Even just copying and pasting the two lines from https://modelviewer.dev/ into the body of my HTML code and not editing them (i.e. not swapping in my model) does nothing.

**Reply by elalish** [reply]
You'll have to start by sharing a link to your website and sharing any errors/warnings you see in your JS console. What kinds of websites have you made so far? `<model-viewer>` is not the simplest place to start, so you may want to begin with a tutorial to make a website with images, videos, and such.

**Reply by ronpadz** [reply]
@JoelTre if your objective is to display a model on your website you can reach out to me at info@nvzn.net and I'll give you access to our platform where you can upload your model then add our iframe player to your site with an embed code.
